presto-product-tests/src/main/java/io/prestosql/tests/hive/TestSqlStandardAccessControlChecks.java

```diff
@@ -240,4 +240,13 @@ public void testAccessControlFilterColumns()
         aliceExecutor.executeQuery(format("GRANT SELECT ON %s TO bob", tableName));
         assertThat(bobExecutor.executeQuery(format("SELECT * FROM information_schema.columns WHERE table_name = '%s'", tableName))).hasRowsCount(2);
     }
+
+    @Test(groups = {AUTHORIZATION, PROFILE_SPECIFIC_TESTS})
+    public void testHiveAlterPrivilege()
+    {
+        onHive().executeQuery(format("GRANT ALTER ON TABLE %s TO USER bob", tableName));
```

Comment:
I think we don't need these two words: TABLE, USER
examples here: https://prestosql.io/docs/333/sql/grant.html#examples
`GRANT INSERT, SELECT ON orders TO alice;`